Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/state/ThresholdedRandomCutForestState.java

```diff
@@ -46,14 +52,17 @@ public class ThresholdedRandomCutForestState implements Serializable {
     private boolean inHighScoreRegion;
     private boolean ignoreSimilar;
     private int numberOfAttributors;
+    // end deprecated segment
 
     private long randomSeed;
 
     private String forestMode;
     private String transformMethod;
+    private String scoringStrategy;
+    @Deprecated
```

Comment:
Would add the notation bring issues? I remember you told me you can only add comments. Even adding an annotation would cause protostuff to not recognize the bytes.

Reply:
It is possible that protostuff causes an issue -- there are probably settings we could explore, but it may be safer/simpler to remove this. Removed. We should have large scale cleanup in the 3.x -> 4.0 jump for the library.